packages/core/src/routes/application.ts

```diff
@@ -27,9 +32,14 @@ export default function applicationRoutes<T extends AuthedRouter>(router: T) {
       findAllApplications(limit, offset),
     ]);
 
+    const oidcConfig = await got(discoveryUrl).json<SnakeCaseOidcConfig>();
```

Comment:
Can we fetch data from `oidc-provider` or some `node` native funtions?

Reply:
No, I have researched.